client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -89,10 +89,18 @@ struct SubscribeConfig {
   std::string id;
   // If set to false then all certificates related to subscription will be ignored
   bool use_tls;
+  // TRC/S certificate path
+  std::string trsc_tls_cert_path;
```

Comment:
Why do we need this here? You only need the path when you configure the client and read the certs.

Reply:
Thanks for catching, updated.